index-vectors.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DATA_DIR="./elasticsearch/data"
```

Comment:
`mkdir -p DATA_DIR` would make this folder as needed.

Reply:
Done